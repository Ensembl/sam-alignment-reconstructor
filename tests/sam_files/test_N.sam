r004	0	ref	16	30	6M14N1I5M	*	0	0	ATAGCTCTCAGC	*	NM:i:1	MD:Z:11
