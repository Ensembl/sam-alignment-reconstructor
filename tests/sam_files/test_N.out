r004
ATAGCT..............CTCAGC
||||||               |||||
ATAGCT..............-TCAGC

